docs/src/goals.rst

```diff
@@ -76,48 +74,60 @@ the metatensor ecosystem!
-prototyping tool for new models. While it is possible to use metatensor to only
-exchange data between libraries (and immediately convert everything to library
-specific format); we also provide tools to operate on metatensor data, staying in
-the metatensor format.
+.. warning::
 
-We call these tools :ref:`operations <python-api-operations>`, and they
-available in the Python interface to metatensor. By using combining multiple
-operations, you can build custom machine learning models, using data and
-representations coming from arbitrary metatensor-compatible libraires in the
-ecosystem. Using these operations allow you to keep your data in metatensor
-format across the whole ML pipeline; meaning the metadata is kept up to date
-with the data, and arbitrary gradients are automatically updated to stay
-consistent with the values.
+    The learning utilities are still an early work in progress, with a lot more
+    building blocks to be included.
 
 .. table:: Where similar functionalities is provided by different packages
     :widths: auto
 
-    +-------------+----------------------------------+---------------------------+--------------------------------------+
-    |  Package    | Core data class                  |  Operations               |  Machine learning models facilities  |
-    +=============+==================================+===========================+======================================+
-    |  numpy      | :py:class:`numpy.ndarray`        | :py:func:`numpy.sum`      |  `scikit-learn`_                     |
-    +-------------+----------------------------------+---------------------------+--------------------------------------+
-    |  torch      | :py:class:`torch.Tensor`         | :py:func:`torch.add`      | :py:class:`torch.nn.Module`,         |
-    |             |                                  |                           | :py:class:`torch.utils.data.Dataset` |
-    +-------------+----------------------------------+---------------------------+--------------------------------------+
-    |  metatensor | :py:class:`metatensor.TensorMap` | :py:func:`metatensor.add` | Work in progress, will be part of    |
-    |             |                                  |                           | the ``metatensor-learn`` package     |
-    +-------------+----------------------------------+---------------------------+--------------------------------------+
+    +-------------+----------------------------------+---------------------------+----------------------------------------------+
+    |  Package    | Core data class                  |  Operations               |  Machine learning models facilities          |
+    +=============+==================================+===========================+==============================================+
+    |  numpy      | :py:class:`numpy.ndarray`        | :py:func:`numpy.sum`      |  `scikit-learn`_                             |
```

Comment:
is `numpy.sum` the same as `torch.add`? I think `sum` is adding up array elements, `add` is summing two arrays.

Reply:
not really, I think I used it because we have `metatensor.add` but no `np.add`. I might change that to `pow` which is everywhere